composer.json

```diff
@@ -1,26 +0,0 @@
-{
-    "name": "phpro/symfony-conventions",
-    "description": "The basic style rules",
-    "type": "library",
-    "license": "propriatary",
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "require": {
-        "brianium/paratest": "*",
-        "friendsofphp/php-cs-fixer": "^2.16",
-        "nikic/php-parser": "^4.3",
-        "php-parallel-lint/php-parallel-lint": "^1.2",
-        "phpro/grumphp": "^1.3",
-        "phpstan/phpstan": "^0.12.3",
```

Comment:
I've been using psalm in our latest 3-4 projects. Maybe we can discuss which one we pick?
It's mostly the advanced typing system in psalm that attracts me, even though phpstan has similar features.

Reply:
Psalm requires you to learn a bit more i think (their annotations & all) while phpstan is more of a drop-in that just checks for fuckups.